corehq/apps/userreports/util.py

```diff
@@ -172,3 +173,16 @@ def get_backend_id(config, can_handle_laboratory=False):
     if settings.OVERRIDE_UCR_BACKEND:
         return settings.OVERRIDE_UCR_BACKEND
     return config.backend_id
+
+
+@contextmanager
+def temp_setting_value(name, value):
```

Comment:
@NoahCarnahan @snopoke Do we have something similar somewhere else? And is the best place to put this (if we're only using it in `userreports.views`)?